EIPS/eip-5050.md

```diff
@@ -0,0 +1,351 @@
+---
+eip: 5050
+title: Interactive NFTs with Modular Environments
+description: Action messaging and discovery protocol for interactions on and between NFTs
+author: Alexi (@0xalxi)
```

Comment:
```suggestion
author: Alexi (@alexi)
```
Typo

Reply:
oh I was using my twitter handle @0xalxi is that not right?

Reply:
It's the GitHub handle that the bot uses.